src/lib/chainNode.ts

```diff
@@ -150,7 +152,18 @@ export default class ChainNode {
     this.logger.debug('Preparing Transaction inputs: %j outputs: %j', inputs, fulfilledOutputs)
 
     await this.api.isReady
-    const extrinsic = this.api.tx.utxoNFT.runProcess(process, inputs, fulfilledOutputs)
+    //optionally use proxy here
+
+    let extrinsic
```

Comment:
This is an implicit any. I'm very surprised typescript isn't complaining. Please put a type on this. I also recommend removing the `else` clause below this by changing the logic to be more like:

```ts
let extrinsic: SomeType = this.api.tx.utxoNFT.runProcess(process, inputs, fulfilledOutputs)
if (this.proxyAddress) {
  extrinsinc =  this.api.tx.proxy.proxy({ id: this.proxyAddress }, null, extrinsic)
}
```